apps/wallet/src/engine/runners/solana/history/index.ts

```diff
@@ -160,7 +160,9 @@ const constructTransactionHistoryDocument = async (
 		date,
 		status,
 		...balances,
-	};
+	} as HistoryDocument<
```

Comment:
use `satisfy HistoryDocument<SolanaTransferHistory | SolanaSwapHistory | SolanaUnknownHistory
	>;` on this kind of case

Reply:
I have just searched for `sastifies` vs `as`. `sastifies` is really better for type checking (actually be able to check be ts compiler) than `as` (we are responsible for type converting)

Reply:
I resolved by removing `as`, and also not use `satifies`. I think `satifies` stand for type definition of an object, and it was mentioned as return type of function which it is returned. Result actual type is much better for type safety instead of type assertion